.github/scripts/lgtm_bot_diagnose.py

```
@@ -0,0 +1,346 @@
+
+def run_classifier(repo: str, sha: str) -> list[dict]:
+    """Run the classifier script and capture JSON output."""
+    script = os.path.join(os.path.dirname(__file__), "classify_workflow_errors.py")
+    env = {
+        **os.environ,
+        "GITHUB_REPOSITORY": repo,
+        "COMMIT_SHA": sha,
+        "OUTPUT_ONLY": "true",
+    }
+    try:
+        r = subprocess.run(
```

Ruff (0.15.0)

[error] `subprocess` call: check for execution of untrusted input

(S603)